content/concepts/asset-manager.md

```diff
@@ -1963,6 +1963,12 @@ An asset can hold one or several files. This comes in pretty handy if, for insta
 }
 ```
 
+::: info
+When you are searching for the information of an asset on a product, if you do not know the asset familly code, you can retreive it by calling this endpoint `/api/rest/v1/attributes/YourLinkedAssetCollectionAttribute`.
```

Comment:
Maybe, precise what `YourLinkedAssetCollectionAttribute` is. So that readers don't copy/paste the link blindly without making changes. ;)